lib/Sema/CSDiag.cpp

```diff
@@ -6246,6 +6282,35 @@ bool FailureDiagnosis::visitApplyExpr(ApplyExpr *callExpr) {
 
   auto fnType = getFuncType(CS->getType(fnExpr));
 
+  // Let's see if this has to do with member vs. property error
+  // because sometimes when there is a member and a property declared
+  // on the nominal type with the same name type-checking function
```

Comment:
I think this was meant to end t "same name." Followed by "Type-checking the function expression...".